core/templates/components/forms/schema-based-editors/schema-based-editor.component.ts

```diff
@@ -65,7 +65,7 @@ implements AfterViewInit, ControlValueAccessor, Validator {
   }
 
   @Output() localValueChange = new EventEmitter();
-  constructor(private elementRef: ElementRef) { }
+  constructor(public elementRef: ElementRef) { }
```

Comment:
Why this change?

Reply:
This is not needed as a different approach is being used to manage the height of the editor. Removed.